src/libcore/cmp.rs

```diff
@@ -96,6 +96,7 @@ use self::Ordering::*;
 /// For example, let's tweak our previous code a bit:
 ///
 /// ```
+/// #[derive(PartialEq)]
```

Comment:
While we're explaining everything, i wonder if we can include a comment on this line like "`// Implements <BookFormat> == <BookFormat> comparisons`", just to hammer it home.

Reply:
done in https://github.com/rust-lang/rust/pull/57357/commits/423a5bb5c437c2ce81d4150490d0cccfdbc33a4c